lib/ansible/modules/windows/win_format.py

```diff
@@ -70,6 +70,8 @@
         or if the source and target file system are different.
     type: bool
 notes:
+  - A minimum Operating System Version of 6.2 is required to use this module. To check if your OS is compatible, see
```

Comment:
Instead of saying the OS version, just say that this module requires Server 2012/Windows 8 or newer. No need to confuse people with OS version strings.

Reply:
@jborean93 Updated to specify those products